pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -356,15 +359,12 @@ func createVault(vaultPath string) error {
 	return linkKeyrings()
 }
 
-func setupVault(vaultPath string) error {
-	if _, err := os.Stat(vaultPath); os.IsNotExist(err) {
+func setupVault(vaultPath string, deprecated bool) error {
```

Comment:
Would be helpful to document that when deprecated is set it doesn't create the vaultPath directory